### PR TITLE
prepare release of 0.3.0 (#299)

### LINKED ISSUE #299: release plugin 0.3.0
With the new feature where one can edit local files and push them to the cluster time has come to release the plugin 

depends on #293, #294, #295, #298, #301, #303

### PR DESCRIPTION
**Pull request by adietish**
fixes #299 
should be merged after: #301

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -11,7 +11,13 @@
     <change-notes>
         <![CDATA[
         <p></p>
-        <h3>0.2.3</h3>
+        <h3>0.3.0</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/217">Resource editor: should allow to edit files loaded from local disk (File > Open)</a>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/266">Support Kubernetes 1.22.0</a>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/281">Pods for a deployment are not listed. An error is displayed instead.</a>
```
Comment (lstocchi) [inline src/main/resources/META-INF/plugin.xml:18]: Missing `</li>` at the end

**Review by jeffmaury — CHANGES_REQUESTED** [review]
Telemetry addition is not mentioned in the plugin.xml.
Also noted that the link from README to USAGE_DATA is broken

**Comment by adietish** [comment]
@lstocchi, @jeffmaury: thx for pointing out. I fixed those issues and rebased. Please re-review.

**Review by lstocchi — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
corrected missing `</li>` in `plugin.xml` and rebased.

**Pull request merged**